### PR TITLE
gh-2557 Added calls to get more data

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Handles #2557

Changes proposed in this pull request:
- New api call
- Refactor the namesof methods to be more swifty

### CODE REVIEW

File: Multisig/Data/Services/Safe Claiming Service/AllocationsRequest.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    var httpMethod: String { "GET" }
+
+    var urlPath: String {
+        "/claiming-app-data/resources/data/allocations/\(account.hexadecimal).json"
+    }
+
+    typealias ResponseType = [Allocation]
+
+    func encode(to encoder: Encoder) throws {
```
Comment (MouazAlzahabi) [inline Multisig/Data/Services/Safe Claiming Service/AllocationsRequest.swift:22]: Should we have this method ?

File: Multisig/Data/Services/Safe Claiming Service/SafeClaimingService.swift [hunk 2]
```diff
@@ -19,6 +19,7 @@ class SafeClaimingService {
         self.url = URL(string: "https://5afe.github.io/")!
         httpClient = JSONHTTPClient(url: url, logger: logger)
         httpClient.jsonDecoder.dateDecodingStrategy = .millisecondsSince1970
+        httpClient.jsonDecoder.keyDecodingStrategy = .convertFromSnakeCase
```
Comment (MouazAlzahabi) [inline Multisig/Data/Services/Safe Claiming Service/SafeClaimingService.swift]: Isn't it better if we have this on the source files directly from now?

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/Data/Services/Safe Claiming Service/AllocationsRequest.swift (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig/Data/Services/Safe Claiming Service/AllocationsRequest.swift:22]: needed to implement encodable

File: Multisig/Data/Services/Safe Claiming Service/SafeClaimingService.swift (hunk 2 above)
Comment (DmitryBespalov) [reply Multisig/Data/Services/Safe Claiming Service/SafeClaimingService.swift]: ok, will update the data

**Pull request merged**